main.js

```diff
@@ -0,0 +1,25 @@
+const ul = document.getElementById("js-target"); 
+const fragment = document.createDocumentFragment(); 
+const elements = [ 
+  {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, 
+  {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"} 
+]; 
+
+elements.forEach(function(e){ 

```

Comment:
IMO
It is a good practice to use a singular form of the array name instead of 'e'.

```suggestion
elements.forEach(function(element){
```

Reply:
Thank you saki-san :)
I see.
There isn't any events so e hasn't meaning .
I got it.